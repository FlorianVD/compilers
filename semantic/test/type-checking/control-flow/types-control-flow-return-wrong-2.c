void foo()
{
    return 2.0;
}
