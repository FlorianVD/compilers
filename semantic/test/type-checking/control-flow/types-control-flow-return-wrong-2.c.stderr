sema: error: Type of return statement does not match function return type
